src/test/java/com/blog/dao/impl/CommentDaoRestClientImplTest.java

```diff
@@ -0,0 +1,388 @@
+package com.blog.dao.impl;
+
+import com.blog.config.SpringTestConfig;
+import com.blog.dao.impl.CommentDaoRestClientImpl;
+import com.blog.model.Comment;
+import com.blog.model.CommentListWrapper;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+@ContextConfiguration(classes = SpringTestConfig.class)
```

Comment:
What for? `@Mock` annotation creates mocks for you, so there're no needs to inject additional configuration class. 
Think it will work w/o it cause you even don't use `SpringJUnit4ClassRunner` or something similar

Reply:
yes, you're right, i forgot to fix it

Reply:
So, you still have `SpringTestConfig` class.
What for? Remove it.
And try to squash these 2 commits in one. Great opportunity to learn this feature of Git.

Reply:
##### SpringTestConfig class already removed from project
![screenshot from 2019-02-24 15-15-12](https://user-images.githubusercontent.com/32134385/53299071-03da9180-3847-11e9-9d57-7d1147a38d50.png)

Reply:
I've missed that class has been removed, sorry.
Squash all of that and make just one commit

Reply:
Done